GeosCore/emissions_mod.F90

```diff
@@ -388,4 +388,235 @@ SUBROUTINE Emissions_Final( Error, RC )
     ENDIF
   END SUBROUTINE Emissions_Final
 !EOC
+!------------------------------------------------------------------------------
```

Comment:
This routine was deleted in https://github.com/geoschem/geos-chem/pull/1816. It must have snuck back in with a merge. Please remove again.